packages/lab/src/components/Flow/hooks/useFlowNode.ts

```diff
@@ -1,24 +1,53 @@
-import { Edge, Node, useEdges, useNodes } from "reactflow";
+import { useMemo } from "react";
+import { Node, Edge, ReactFlowState, useStore } from "reactflow";
 
-export function useFlowNode<T = any>(id: string) {
-  const nodes = useNodes<T>();
-  const edges = useEdges();
+export function useFlowNode<T extends Node = Node>(
+  id: string
+): Node | undefined {
```

Comment:
```suggestion
 {
```
By casting the return value to `Node`, we're losing the generic "feature". TS's inference already gives us the right result